script.js

```diff
@@ -1 +1,189 @@
-//CEOS 19기 프론트엔드 파이팅🔥 ദ്ദി˶ˊᵕˋ˵)
+/*
+  checkboxNumber: todoCard가 unique한 id를 갖기 위한 변수
+*/
+let checkboxNumber = 0;
+
+//todoCard 만들기
+const createTodoCard = (todo, isDone) => {
+  const todoCard = document.createElement('div');
+  const checkbox = document.createElement('input');
+  const todoContent = document.createElement('label');
+  const todoDelete = document.createElement('div');
+
+  todoCard.className = 'todoCard';
+  checkbox.type = 'checkbox';
+  checkbox.className = 'todoCheckbox';
+  checkbox.id = `todoCheckbox${checkboxNumber}`;
+  if (isDone) {
+    checkbox.checked = true;
+    todoCard.className = 'doneCard';
+  } else {
+    todoCard.className = 'todoCard';
+  }
+  todoContent.className = 'todoContent';
+  todoContent.htmlFor = `todoCheckbox${checkboxNumber++}`;
+  todoContent.textContent = todo;
+  todoDelete.className = 'todoDelete';
+  todoDelete.textContent = 'X';
+
+  todoCard.append(checkbox, todoContent, todoDelete);
+  if (isDone) {
+    document.querySelector('.doneSection').append(todoCard);
+  } else {
+    document.querySelector('.todoSection').append(todoCard);
+  }
+};
+
+//localstorage에 저장된 todo,done 불러오기
+const todoList = JSON.parse(localStorage.getItem('todo'));
+const donelist = JSON.parse(localStorage.getItem('done'));
```

Comment:
여기서 localStorage.getItem('todo')와 localStorage.getItem('done') 이 null인 경우를 방지하기 위해 이런식으로 코드 리팩토링하셔도 좋을 것 같습니다~
```suggestion
const doneList = JSON.parse(localStorage.getItem('done'), '[]');
```